packages/solutions/app-tools/src/builder/shared/bundlerPlugins/RouterPlugin.ts

```diff
@@ -155,7 +155,7 @@ export class RouterPlugin {
             if (!asset) {
               continue;
             }
-            const newContent = `${injectedContent}${asset.source().toString()}`;
+            const newContent = `${asset.source().toString()}${injectedContent}`;
```

Comment:
If the `injectedContent` is placed in the post-execution here, will there be a problem with the order of code execution?

Reply:
The `injectedContent` will have been executed when runtime plugin accesses it, so it won't break the function.